docker/fullnode/README.md

```diff
@@ -1,83 +1,93 @@
 
-```wget https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml```
+```shell
+wget https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml
+```
+
+TODO: Don't they need to rename this file to `fullnode.yaml` too?
 
-## sui devnet genesis
+### sui devnet genesis
 
-Get the latest version of the Sui DevNet genesis blob [here](https://github.com/MystenLabs/sui-genesis/raw/main/devnet/genesis.blob), or:
+Get the latest version of the Sui DevNet genesis [genesis.blob](https://github.com/MystenLabs/sui-genesis/raw/main/devnet/genesis.blob) file over the web or:
 
 ```wget https://github.com/MystenLabs/sui-genesis/raw/main/devnet/genesis.blob```
 
 
-## start the fullnode
+## Start the fullnode
 
-```docker-compose up```
+To start the fullnode using Docker, run:
 
-# test
+```shell
+docker-compose up
+```
 
-Once the fullnode is up and running, test some of the jsonrpc interfaces.
+## Test
 
-- get the five most recent transactions:
+Once the fullnode is up and running, test some of the JSON-RPC interfaces.
+
+- Get the five most recent transactions:
 
 ```
 curl --location --request POST 'http://127.0.0.1:9000/' \
     --header 'Content-Type: application/json' \
     --data-raw '{ "jsonrpc":"2.0", "id":1, "method":"sui_getRecentTransactions", "params":[5] }'
 ```
 
-- get details about a specific transaction:
+- Get details about a specific transaction:
 
 ```
 curl --location --request POST 'http://127.0.0.1:9000/' \
     --header 'Content-Type: application/json' \
     --data-raw '{ "jsonrpc":"2.0", "id":1, "method":"sui_getTransaction", "params":["$RECENT_TXN_FROM_ABOVE"] }'
 ```
 
-# use your fullnode with explorer 
+## Use your fullnode with Explorer 
 
-To use the DevNet explorer with your fullnode follow these steps:
+To use the Sui Explorer with your fullnode, follow these steps:
 - Open https://explorer.devnet.sui.io
-- Click the green Devnet button in the top right
+- Click the green **Devnet** button in the top right
 - Select 'Custom RPC URL'
 - Set it to http://127.0.0.1:9000
 
-# troubleshoot / tips / documentation
+TODO: Confirm this shouldn't be Local as in the existing fullnode doc.
+
+## Troubleshoot / tips / documentation
 
-## start the fullnode in detached mode
+### Start the fullnode in detached mode
 
 ```docker-compose up -d```
 
-## stop the fullnode:
+### Stop the fullnode
 
 ```docker-compose stop```
 
-## reset the environment
+### Reset the environment
 
 Take everything down, removing the container and volume. Use this to start completely fresh (image, config, or genesis updates):
 
 ```docker-compose down --volumes```
 
-## inspect the state of a running fullnode
+### Inspect the state of a running fullnode
 
-Get the running container id:
+Get the running container ID:
```

Comment:
nit:  'docker ps' shows all the containers running.

Reply:
Should this be added to the doc?

Reply:
no. just a nit.